tests/lunamark/ext_fenced_divs.test

```diff
@@ -66,6 +66,11 @@ This is the beginning of a div
 This is the end of a div
 :::
 
+::: {#another-identifier}
+> This is a blockquote immediately in a
+> div
+:::
+
 Here are some extra colons that will be shown as just text:
 
 ::::
```

Comment:
This does not seem as an ideal placement for the test case, because the trailing `:::` will cause `::: {#another-identifier}` to form a fenced div. [Here][1] seems to be a good placement that reproduces the error reported in #60.

 [1]: https://github.com/Witiko/markdown/pull/259/commits/ad3407bc489ad6d60060e8a4d3efd893086682cf

Reply:
Move earlier as suggested.